### DISCUSSION TITLE
Github copilot is not Available in Github Student Developer Pack

### DISCUSSION THREAD

**Discussion by om2137** [community/community#19079] [Copilot Conversations] [answered]
I am verified student and was trying to search for copilot, but it is not available in student developer pack. on billing page it was mentioned that it was available free for students and selected maintainers.

**Comment by ldeluigi** [comment, marked as answer]
Duplicate of #19050

**Comment by om2137** [comment]
@[ldeluigi](https://github.com/ldeluigi) thankyou for answer, i was not aware of this thread.
